defaults/api_ents.go

```diff
@@ -0,0 +1,112 @@
+package defaults
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/moby/libentitlement/entitlement"
+	"github.com/moby/libentitlement/secprofile"
+)
+
+const (
+	// APIEntFullID is the API entitlement identifier; the value format is: "api.access:api-id:subset:[allow|deny]"
+	// ex: "api.access:engine.swarm:all:allow"
+	APIEntFullID = "api.access"
+
+	// APIFullControl specifies access control for the whole api
+	APIFullControl = "all"
+)
+
+// Default known APIs and API subsets to control access of
+const (
+	// EngineAPI defines the Moby-Engine API
+	EngineAPI = "engine"
+
+	// SwarmAPI defines the Moby-Engine Swarm API
+	SwarmAPI = "swarm"
+)
+
+var (
+	apiEntitlement = entitlement.NewStringEntitlement(APIEntFullID, apiEntitlementEnforce)
+)
+
+// GetSwarmAPIIdentifier returns the full Swarm API identifier
+func GetSwarmAPIIdentifier() secprofile.APIID {
+	return secprofile.APIID(fmt.Sprintf("%s.%s", EngineAPI, SwarmAPI))
+}
+
+/*IsSwarmAPIControlled checks if Moby Swarm API is controlled and whether it's allowed or not
+ * Return values are the following:
+ * isControlled - if no error is encountered, whether the Swarm API is currently controlled by the entitlements
```

Comment:
Hmm I thought `IsSwarmAPIControlled` would make more sense as the default behavior should be controlled by the container management platform.

Follow-up note: we should have a set-able `DefaultAccess` that'd be returned in case of an error.